etna/packages/etna-js/spec/__mocks__/webworkify-webpack.js

```diff
@@ -0,0 +1,48 @@
+import {Debouncer} from "../../utils/debouncer";
+
+export default function mockWebworkify(script) {
```

Comment:
This mock is enabled in `setup.js` and allows us to deep test the worker integration.  It mocks the `webworkify-webpack` module and largely behaves the same way, but performs worker logic in the same thread behind promises.